apps/talk.sh

```diff
@@ -20,6 +18,19 @@ debug_mode
 # Must be root
 root_check
 
+# Check if adminer ist already installed
+print_text_in_color "$ICyan" "Checking if Talk is already installed..."
+if is_this_installed coturn
```

Comment:
Just because coturn is installed it doesn't mean that Talk is activated. Check with `occ` if any settings are made for example. 

[Here's](https://github.com/nextcloud/vm/blob/master/nextcloud_update.sh#L557-L577) and example of a good check.

Reply:
Yes, thats right, but please answer my comment above first.
Afterwards I will improve the checks.

Reply:
should be done now.